.gitmodules

```diff
@@ -1,6 +1,12 @@
 [submodule "libsbp"]
-	path = libsbp
+	path = third_party/libsbp
 	url = https://github.com/swift-nav/libsbp.git
 [submodule "libswiftnav"]
-	path = libswiftnav
+	path = third_party/libswiftnav
 	url = https://github.com/swift-nav/libswiftnav.git
+[submodule "common-cmake"]
+	path = cmake/common
+	url = git@github.com:swift-nav/cmake.git
```

Comment:
Change to a https URL to allow cloning without key.

Reply:
Done